src/stp/skills/GoToPos.cpp

```diff
@@ -16,13 +16,21 @@ Status GoToPos::onUpdate(const StpInfo &info) noexcept {
         targetPos = control::ControlUtils::projectPositionToWithinField(info.getField().value(), targetPos, control_constants::ROBOT_RADIUS);
     }
 
+    std::pair<RobotCommand, std::optional<Vector2>> robotCommandCollisionPair;
     // Calculate commands from path planning and tracking
-    auto robotCommand = info.getCurrentWorld()->getRobotPositionController()->computeAndTrackPath(
+    // _______Use this for the old pathplanning and tracking_______
+//    robotCommandCollisionPair.first = info.getCurrentWorld()->getRobotPositionController()->computeAndTrackPath(
+//        info.getField().value(), info.getRobot().value()->getId(), info.getRobot().value()->getPos(), info.getRobot().value()->getVel(), targetPos, info.getPidType().value());
+    // _______Use this one for the BBT pathplanning_______
+    robotCommandCollisionPair = info.getCurrentWorld()->getRobotPositionController()->computeAndTrackPathBBT(
         info.getField().value(), info.getRobot().value()->getId(), info.getRobot().value()->getPos(), info.getRobot().value()->getVel(), targetPos, info.getPidType().value());
```

Comment:
No commented code; prefer writing something like this instead;

bool use_old_pathplanning = false;
if(use_old_pathplanning){
... (old code)...
}else{
... (new code) ...
}

Reply:
This way, we can be sure the old code also still compiles.